google/cloud/forseti/scanner/audit/lien_rules_engine.py

```diff
@@ -197,6 +197,10 @@ def find_violations(self, parent_resource, liens):
 
         for res in resource_ancestors:
             applicable_rules.extend(self.resource_to_rules.get(res, []))
+            wildcard_res = resource_util.create_resource(
```

Comment:
if we put this in the loop, will the same rule gets added multiple times to applicable_rules?